syncto/tests/test_authentication.py

```diff
@@ -57,7 +57,6 @@ def test_should_cache_credentials_the_second_time(self):
                 build_sync_client(self.request)
                 SyncClient.assert_called_with(**self.credentials)
                 # Second time
-                self.request.headers[CLIENT_STATE_HEADER] = '6789'
```

Comment:
Now that we use the Client-State to encrypt the data, it should not change in between calls.